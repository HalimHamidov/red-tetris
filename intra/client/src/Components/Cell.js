import React from "react";
import { Tetrominos } from "../tetrominos";
import { StyledCell } from "./styling/StyledCell";

const Cell = ({ type }) => (
  <StyledCell type={type} color={Tetrominos[type]?.color}></StyledCell>
);

export default React.memo(Cell);
