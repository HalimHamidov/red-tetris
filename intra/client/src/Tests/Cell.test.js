import renderer from "react-test-renderer";
import Cell from "../Components/Cell";

test("App react-test-renderer test ", () => {
    const cellMatch = renderer.create( < Cell / > ).toJSON();
    expect(cellMatch).toMatchSnapshot();
});