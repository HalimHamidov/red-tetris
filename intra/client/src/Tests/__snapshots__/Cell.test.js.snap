// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App react-test-renderer test  1`] = `
<div
  className="sc-bdvvtL KhUZD"
/>
`;
